content/kubernetes/release-notes/k8s-6-2-18-41-jan-2023.md

```diff
@@ -17,18 +17,24 @@ New images and fixes are listed below. Refer to [6.2.18-41 (Dec 2022]({{<relref
 
 ## Images
 
+DockerHub images are available at `docker.io/`.
+
 * **Redis Enterprise**: `redislabs/redis:6.2.18-72`
 * **Operator**: `redislabs/operator:6.2.18-41`
 * **Services Rigger**: `redislabs/k8s-controller:6.2.18-41`
 
 ### OpenShift images
 
-* **Redis Enterprise**: `redislabs/redis:6.2.18-72.rhel8-openshift` (or `redislabs/redis:6.2.18-72.rhel7-openshift` if upgrading from RHEL 7)
-* **Operator**: `redislabs/operator:6.2.18-41`
+**OLM operator version**: `v6.2.18-41b`
+
+Red Hat images are available at `registry.connect.redhat.com/`.
```

Comment:
IMO this would be more usable if we just list the full images below (e.g., `registry.connect.redhat.com/redislabs/redis-enterprise:6.2.18-72.rhel8-openshift`). Ultimately, customers may need to copy-paste this string into their deployment manifests, and it's easier to copy-paste one string rather than stitch two together.